### DISCUSSION TITLE
Express example bug: OAuth (Google) consent screen is shown for every sign-ins

### DISCUSSION THREAD

**Discussion by lihebi** [nextauthjs/next-auth#11919] [Help]
In the [official express example](https://github.com/nextauthjs/next-auth/tree/main/apps/examples/express), the sign-in and sign-out are handled by the REST api `/api/auth/signin` and `/api/auth/signout`.

When a user sign-in with google, the consent screen is displayed every single time. That is, when user signout and sign-in again, the consent screen is shown again.

The expected behavior is that the user only need to agree to consent screen for the first time login to this site. This behavior is correct in the [official nextjs example](https://github.com/nextauthjs/next-auth/tree/main/apps/examples/nextjs).

This is the consent screen:

<img width="300" alt="Screenshot 2024-09-27 at 4 28 10 PM" src="https://github.com/user-attachments/assets/def63f67-71f5-47c9-8363-65c2a855b9bf">

**Comment by ygorluiz** [comment]
@lihebi Did you find a solution ?

**Comment by lihebi** [comment]
No, I couldn't find a solution yet.
